### PR TITLE
#5295 [Manager] The APIs used to verify should be called after the ex…

### PR DESCRIPTION
**Pull request by eechih**
…ecution APIs - worker

### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue #5295 
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/observables/__tests__/workers.test.js [hunk 1]
```diff
@@ -78,15 +79,21 @@ it('start worker should run in two minutes', () => {
...
+      of(RESPONSES.successWithRunning).pipe(delay(100)),
+    );
 
     const params = {
       group: workerEntity.group,
       name: workerEntity.name,
     };
 
-    const expected = '10s 10ms (v|)';
+    // stop 6 times, get 6 times, retry 5 times
```
Comment (saivirtue) [inline ohara-manager/client/src/observables/__tests__/workers.test.js:94]: should be `start 6 times, get 6....`

File: ohara-manager/client/src/observables/workers.ts [hunk 2]
```diff
@@ -31,20 +31,19 @@ export function fetchWorker(values: ObjectKey) {
...
 }
 
+// Attempt to start at intervals and wait until the service is not running
 export function startWorker(key: ObjectKey) {
-  return zip(
-    // attempt to start at intervals
+  return of(
     defer(() => workerApi.start(key)),
-    // wait until the service is running
     defer(() => workerApi.get(key)).pipe(
```
Comment (saivirtue) [inline ohara-manager/client/src/observables/workers.ts:38]: Could `of(a, b)` guarantee `b` will always be triggered after `a` emit value?

Comment (eechih) [reply ohara-manager/client/src/observables/workers.ts:38]: Sure

**Comment by eechih** [comment]
retry manager-e2e

**Pull request merged**